src/common/components/status-tag/status-tag.component.tsx

```diff
@@ -35,17 +45,30 @@ export class StatusTag extends Component<StatusTagProps> {
         }
     };
 
+    mapSizeToStyle = (size: StatusTagSize): CSSProperties => {
+        switch (size) {
+            case 'small':
+                return { padding: '0px 6px 10px', fontSize: 10 };
+            case 'large':
+                return { padding: '6px 9px 24px', fontSize: 15 };
+            default:
+            case 'medium':
+                return { padding: '3px 7px 22px', fontSize: 13 };
+        }
+    };
+
     render() {
-        const { status } = this.props;
+        const { status, size, extended } = this.props;
 
         const color = mapStatusToColor(status);
         const tooltip = this.mapStatusToTooltip(status);
         const icon = this.mapStatusToIcon(status);
 
         return (
             <Tooltip placement="top" title={tooltip}>
-                <Tag color={color}>
+                <Tag style={this.mapSizeToStyle(size!)} color={color}>
                     <Icon type={icon} />
+                    {extended ? <span style={{ paddingLeft: '5px' }}>{status}</span> : null}
```

Comment:
I thought we were doing this with `className` and exernal less files, any reason not to use this here?

Reply:
I felt like this was appropriate for one property, which is just adding some spacing between the icon and the status. Using `className` with an external less file would have made it less neat. This can be fixed in the future if we have more styling.